sourcegraph/sourcegraph
# bext: fix Sourcegraph button styles on Bitbucket

Closes: https://github.com/sourcegraph/sourcegraph/issues/32598

| Before | After |
| --------- | ------ |
| ![image](https://user-images.githubusercontent.com/25318659/162975751-911e10c1-f04b-4c8c-863a-679cc0210301.png) | ![Screenshot](https://user-images.githubusercontent.com/25318659/162975898-7888bcc3-c80d-4368-96a2-f410959c5ce1.png) |

## Test plan
- `sg run bext`
- open not added to Sourcegraph instance repo page
- check that Sourcegraph button has similar styles as code host buttons 


## App preview:

- [Link](https://sg-web-taras-yemets-fix-sourcegraph.onrender.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
